requirements.yml

```diff
@@ -9,3 +9,4 @@ roles:
 
 collections:
   - name: openstack.cloud
+    version: 1.10.0
```

Comment:
Should we merge in the upstream repo to get the fix for this: https://github.com/stackhpc/openstack-config/commit/a09540f5c84b8055e09b74bf84d965894c47704d?

Reply:
Yep, that's probably a better solution